snuba/settings/settings_test.py

```diff
@@ -22,6 +22,11 @@
 # to explore the unrefined Expression structure
 PRETTY_FORMAT_EXPRESSIONS = True
 
+# For converting code storages/entities/datasets to yaml we need to record what
+# arguments certain classes were instantiated with. This has no purpose in a production
+# environment. Thus it is only enabled here
+ENABLE_INIT_RECORDER = True
```

Comment:
Does this even need to be enabled here? It's possible to override settings on the CLI (`ENABLE_INIT_RECORDER=1 snuba translate-dataset ...`).

Reply:
TIL, I'll remove it from here then!